ossapi/ossapiv2.py

```diff
@@ -449,6 +449,25 @@ def beatmap(self, beatmap_id: int) -> Beatmap:
         """
         return self._get(Beatmap, f"/beatmaps/{beatmap_id}")
 
+    # /beatmapsets
+    # ____________
+
+    @request
+    def beatmapset_discussion_posts(self,
+        beatmapset_session_id: int = None,
+        limit: int = None,
+        page: int = None,
+        sort: SortOrder = None,
+        user: int = None,
+        with_deleted: bool = None
```

Comment:
these should all be annotated as optional

Reply:
Hmm, okay. This didn't break because apparently the get_type_hints automatically wraps anything with a default value of `None` with `Optional`.

Reply:
you're right that we supply `None` to any *models* with fields annotated as optional, but this is a method and isn't affected by that. These type hints are purely for consumers' benefit.